src/test/resources/org/mule/tools/devkit/sonar/checks/pom/ScopeProvidedInMuleDependenciesCheckTest-pom.xml

```diff
@@ -18,6 +18,7 @@
             <groupId>org.mule.tools.devkit</groupId>
             <artifactId>mule-devkit-annotations</artifactId>
             <version>3.7.2</version>
+            <scope>compile</scope>
```

Comment:
Please revert this. The idea is one pom that checks that there's no explicit scope (so it defaults to compile). And there is another test for explicitly defined compile. This is the former one.